tools/release/helpers/cargo.ts

```diff
@@ -46,3 +46,17 @@ export async function publishCrate(directory: string) {
     throw new Error("Failed");
   }
 }
+
+export async function build(directory: string) {
+  const p = Deno.run({
+    cwd: directory,
+    cmd: ["cargo", "build", "-vv"],
+    stderr: "inherit",
+    stdout: "inherit",
+  });
```

Comment:
I usually use `cargo check` and it works fine for updating the lockfiles but is faster as it doesn't have to emit target binaries

Reply:
Oh, sweet. I'll do that instead (didn't know about it)

Reply:
Done in a91d72d1fc01f0fb6acabfedb510e1a3b4bffdd6

I've left the `build` methods because it might be useful for quickly scripting something on the fly.